im.nheko.Nheko.yaml

```diff
@@ -0,0 +1,247 @@
+      - prefix=/app
+    no-autogen: true
+  - name: cmark
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMARK_TESTS=OFF
+    sources:
+      - sha256: 6c7d2bcaea1433d977d8fed0b55b71c9d045a7cdf616e3cd2dce9007da753db3
+        type: archive
+        url: https://github.com/commonmark/cmark/archive/0.30.2.tar.gz
+  - name: spdlog
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DSPDLOG_BUILD_EXAMPLES=0
+      - -DSPDLOG_BUILD_BENCH=0
+      - -DSPDLOG_BUILD_TESTING=0
+    sources:
+      - sha256: 5197b3147cfcfaa67dd564db7b878e4a4b3d9f3443801722b3915cdeced656cb
+        type: archive
+        url: https://github.com/gabime/spdlog/archive/v1.8.1.tar.gz
+  - name: olm
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    buildsystem: cmake-ninja
+    sources:
+      - commit: 92769cec711c604a1f682b95d6944578d2a1bb3d
+        disable-shallow-clone: true
+        tag: 3.2.12
+        type: git
+        url: https://gitlab.matrix.org/matrix-org/olm.git
+  - name: libsecret
+    buildsystem: meson
+    config-opts:
+      - -Dmanpage=false
+      - -Dvapi=false
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+      # https://gitlab.gnome.org/GNOME/libsecret/-/issues/49
+      - -Dgcrypt=false
+    sources:
+      - commit: 3fe635e64efd4b8dbc9ec3548b0bc8034c7665c4
+        tag: 0.20.4
+        type: git
+        url: https://gitlab.gnome.org/GNOME/libsecret.git
+  #- config-opts:
+  #    - -DCMAKE_BUILD_TYPE=Release
+  #    - -DAVIF_CODEC_AOM=ON
+  #    #- -DBUILD_SHARED_LIBS=OFF
+  #  buildsystem: cmake-ninja
+  #  name: libavif
+  #  sources:
+  #    - sha256: 66e82854ceb84a3e542bc140a343bc90e56c68f3ecb4fff63e636c136ed9a05e
+  #      type: archive
+  #      url: https://github.com/AOMediaCodec/libavif/archive/refs/tags/v0.10.1.tar.gz
+  #- config-opts:
+  #    - -DCMAKE_BUILD_TYPE=Release
+  #    - -DWITH_EXAMPLES=OFF
+  #    #- -DBUILD_SHARED_LIBS=OFF
+  #  buildsystem: cmake-ninja
+  #  name: libheif
+  #  sources:
+  #    - sha256: e1ac2abb354fdc8ccdca71363ebad7503ad731c84022cf460837f0839e171718
+  #      type: archive
+  #      url: https://github.com/strukturag/libheif/releases/download/v1.12.0/libheif-1.12.0.tar.gz
+  #- config-opts:
+  #    - -DCMAKE_BUILD_TYPE=Release
+  #    - -DKIMAGEFORMATS_HEIF=ON
+  #  buildsystem: cmake-ninja
+  #  name: KImageFormats
+  #  sources:
+  #    - commit: ae6b724824fc2fdf71d50dc7ae0052ad1551b25a
+  #      tag: v5.93.0
+  #      type: git
+  #      url: https://invent.kde.org/frameworks/kimageformats.git
+  - name: QtKeychain
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TEST_APPLICATION=OFF
+      - -DQTKEYCHAIN_STATIC=ON
+      - -DBUILD_WITH_QT6=ON
+    buildsystem: cmake-ninja
+    sources:
+      - commit: 69f993c47efed7e557d79a30a367014d9a27d809
+        tag: 0.14.1
+        type: git
+        url: https://github.com/frankosterfeld/qtkeychain.git
```

Comment:
whatever formatter you use should be reconfigure to not order by alphabetical order.

Reply:
That's not a formatter, that's just what we historically used. This file used to be the one for io.github.NhekoReborn.Nheko and we just adapted it to the new version and packages without reordering all fields. I don't remember why that order was chosen years ago, but there seem to be no rules for how to order things for flathub packages?

Reply:
Actually, the order is probably from when we migrated from json to yaml.